### ISSUE TITLE
0.1.1 osx binary is an ELF

### ISSUE THREAD

**Issue by brennantaylor** [nbedos/citop#4]
I don't think that's right

```
~/Downloads/citop-0.1.1-osx-amd64
➜ ./citop --help
Failed to execute process './citop'. Reason:
exec: Exec format error
The file './citop' is marked as an executable but could not be run by the operating system.

~/Downloads/citop-0.1.1-osx-amd64
➜ file citop
citop: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, not stripped
```

**Comment by nbedos** [comment]
Thanks for reporting this. It should be fixed in [version 0.1.2](https://github.com/nbedos/citop/releases/tag/0.1.2) by bd2596f48417215f8a55c9fd9094346e3471320c:
```
$ file citop-0.1.2-darwin-amd64/citop
citop-0.1.2-darwin-amd64/citop: Mach-O 64-bit x86_64 executable
```
Could you please confirm that you can now run the executable?

**Comment by brennantaylor** [comment]
lgtm
